### PR TITLE
Add course to database

### PR DESCRIPTION
**Pull request by biel9436**
Added the initial add course page
Added login page
Started the database stuff

### CODE REVIEW

**Review by xion6136 — APPROVED** [review]

File: ClassySchedule/src/main/java/courses/CourseController.java [hunk 1]
```diff
@@ -66,6 +66,18 @@ public void submitData() {
             return;
         }
 
+        // section number validation. Testing it out by printing right now.
+        try {
+            if(section_number.getLength() == 3) {
+                Integer.parseInt(section_number.getText());
+                return;
+            } else {
+                System.out.println("Must be 3 digits number");
```
Comment (biel9436) [inline ClassySchedule/src/main/java/courses/CourseController.java:75]: Should be "Must be a 3-digit number"

File: ClassySchedule/src/main/java/scheduler/SchedulerController.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+        @FXML
+        private TableColumn<?, ?> Thursday;
+
+        @FXML
+        private TableColumn<?, ?> Tuesday;
+
+        @FXML
+        private TableColumn<?, ?> Wednesday;
+
```
Comment (biel9436) [inline ClassySchedule/src/main/java/scheduler/SchedulerController.java:49]: You should implement Initializable into this class like it has been done in the other controllers.

**Comment by biel9436** [comment]
@xion6136 Next time create a separate branch with your changes so we can keep track of what changes have been made and by who.

File: ClassySchedule/src/main/java/resources/fxml/course.fxml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+<?import javafx.scene.control.ComboBox?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.layout.AnchorPane?>
+
+<AnchorPane prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" fx:controller="courses.CourseController">
+   <children>
+      <TextField fx:id="section_number" layoutX="223.0" layoutY="161.0" promptText="section_number" />
+      <TextField fx:id="class_name" layoutX="224.0" layoutY="187.0" promptText="class_name" />
+      <ComboBox fx:id="dept_name" layoutX="223.0" layoutY="213.0" prefWidth="150.0" promptText="dept_name" />
+      <Button fx:id="submit_button" layoutX="374.0" layoutY="213.0" mnemonicParsing="false" onAction="#submitData" text="submit" />
```
Comment (biel9436) [inline ClassySchedule/src/main/java/resources/fxml/course.fxml:13]: This should be changed to Login button,

**Review by elizabethwhalen — APPROVED** [review]

**Pull request merged**